src/containers/restaurantsList/index.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component, Fragment } from "react";
+import { connect } from "react-redux";
+import { push } from "connected-react-router";
+import { routes } from "../../containers/Router";
+import { getAllRestaurants, getRestaurantDetails } from "../../actions/feedRestaurants";
+import styled from "styled-components";
+//MATERIAL-UI CARD
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+
+
+class RestaurantsList extends Component {
```

Comment:
Poderia ser um componente funcional, já que não usa o `this.state` e nem métodos de ciclo de vida do React.